src/utils/convertors.ts

```diff
@@ -543,8 +548,8 @@ export const queryResultToCsv = (
                 rowValues.push(escapeCsvValue(value));
             });
 
-            return rowValues.join(',');
+            return rowValues.join(getCsvSeperator());
         })
         .join('\n');
-    return `sep=,\n${headers}\n${rows}`;
+    return `\n${headers}\n${rows}`;
```

Comment:
I think we have still set `sep=` in order to Excel/other tools understand which separator to use

Reply:
sep= seems to be a excel only thing.  so doesn't give good results on other tools

Reply:
Not really, Mike and I tested it, if the Sep is `;` then Excel detects it properly without the `sep=;` header and at the same time the header is not supported by others (i.e. Apple Numbers or LibreOffice), so we decided to remove it again. Does it work for you without the header @bk201-? We could also introduce an additional setting for it...

Reply:
OK, I've just asked to be sure. Then @mkrueger could you please remove first `\n`

Reply:
did it in the last commit.